screens/Profile/ProfileScreenController.ts

```diff
@@ -15,14 +24,67 @@ export function useProfileScreen({ navigation }: MainRouteProps) {
   const authService = appService.children.get('auth');
   const settingsService = appService.children.get('settings');
 
+  const [alertMsg, setHasAlertMsg] = useState('');
+  const [
+    biometricState,
+    biometricSend,
+    bioService
+  ] = useMachine(biometricsMachine);
+
+  const isEnabledBio:boolean       = useSelector(bioService, selectIsEnabled);
+  const isUnavailableBio:boolean   = useSelector(bioService, selectIsUnvailable);
+  const isSuccessBio:boolean       = useSelector(bioService, selectIsSuccess);
+  const errorMsgBio:string         = useSelector(bioService, selectError);
+  const unEnrolledNoticeBio:string = useSelector(bioService, selectUnenrolledNotice);
```

Comment:
How is this still getting through... I thought `lint-staged` was supposed to fix this. 🤔